app/controllers/tasks_controller.rb

```diff
@@ -18,9 +18,19 @@ def create
     end
   end
 
+  def edit
+    @task = Task.find(params[:id])
+    @week = find_week
+    render partial: 'edit_modal', layout: false
+  end
+
   def update
     @task = Task.find(params[:id])
-    @task.update_attributes task_params
+    if @task.update_attributes title: params[:task][:title]
+      flash[:notice] = "Task updated successfully!"
```

Comment:
@anjali-sharma flash won't be visible in ajax request and we should display errors in this modal